packages/ast-spec/src/declaration/ClassDeclaration/fixtures/_error_/export-missing-name/config.ts

```diff
@@ -0,0 +1,3 @@
+export default {
+  allowInvalidAST: true,
```

Comment:
if this is an error AST - we should probably error and snapshot the error (as that's what these error tests are intended to be for)

sorry - my earlier comment was about turning the errors on before we switched the default to on.

Reply:
Ah, right, that makes more sense.